src/ints/int10_pal.cpp

```diff
@@ -389,18 +389,19 @@ void INT10_SetColorSelect(Bit8u val) {
 		IO_Write(0x3d9,temp);
 	else if (machine == MCH_PCJR) {
 		IO_Read(VGAREG_TDY_RESET); // reset the flipflop
-		switch(vga.mode) {
-		case M_TANDY2:
-			IO_Write(VGAREG_TDY_ADDRESS, 0x11);
-			IO_Write(VGAREG_PCJR_DATA, val&1? 0xf:0);
-			break;
-		case M_TANDY4:
-			for(Bit8u i = 0x11; i < 0x14; i++) {
+		switch (CurMode->mode) {
+		case 4:
+		case 5:
```

Comment:
Ah, initially I wanted to complain about cases switching from enum to number, but now I see the switch is now occuring on `CurMode->mode` and not on `vga.mode`, so it's fine.